openslides/core/config.py

```diff
@@ -37,21 +49,42 @@ def __init__(self) -> None:
         self.config_variables = {}  # type: Dict[str, ConfigVariable]
 
         # Index to get the database id from a given config key
-        self.key_to_id = {}  # type: Dict[str, int]
+        self.key_to_id = None  # type: Optional[Dict[str, int]]
 
     def __getitem__(self, key: str) -> Any:
         """
         Returns the value of the config variable.
         """
-        # Build the key_to_id dict
-        self.save_default_values()
```

Comment:
This method is not called here anymore. Where do you use it instead?

Reply:
It is called after a migration, see: https://github.com/OpenSlides/OpenSlides/blob/32e20aa6708b14f42fbcfcaa1e0792cb20af514b/openslides/core/apps.py#L57 https://github.com/OpenSlides/OpenSlides/blob/32e20aa6708b14f42fbcfcaa1e0792cb20af514b/openslides/core/apps.py#L127

This line was necessary before, because it made sure, that the dict key_to_id was set: https://github.com/OpenSlides/OpenSlides/blob/32e20aa6708b14f42fbcfcaa1e0792cb20af514b/openslides/core/config.py#L189

Now, get_key_to_id() is used, what does the same thing but uses the cache instead of the database.